### PR TITLE
Merge 19 April: add relevant content from beast page only ways page and delete from b…

### PR DESCRIPTION
**Pull request by EmmaFrith**
Add content relating to ways to train from beast page and delete relevant sections from beast page. 

There's still some duplicate content e.g stuff on Now Teach across the 2 pages - but we can address in a separate PR.  

The scope of this PR is simply to be able to remove the 'ways to train' section of the beast page and feel confident content is covered is needed elsewhere.

### CODE REVIEW

File: content/ways-to-train.md [hunk 1]
```diff
@@ -210,16 +216,12 @@ You can apply to them directly.
...
+A career change into teaching will give you the opportunity to use your life, work experience and passion for your subject to inspire young people.
 
-### Researchers in schools
+[Now Teach](https://nowteach.org.uk/) and [Transition to Teach](https://www.transitiontoteach.co.uk/) can support you with your career change.
 
-To go down the [researchers in schools](/guidance/become-a-teacher-in-england#researchers-in-schools-candidates-with-a-doctorate) route you’ll need a doctorate, or be about to receive one, in the subject you want to teach.
+### Researchers in schools
 
-You’ll then train over 3 years with a bursary.
+The [researchers in schools](https://thebrilliantclub.org/researchers-in-schools/) offers a route into teaching if you have or are about to get a PHD.
```
Comment (EmmaFrith) [inline content/ways-to-train.md]:
```suggestion
[Researchers in schools](https://thebrilliantclub.org/researchers-in-schools/) offers a route into teaching if you have or are about to get a PHD.
```

File: content/guidance/become-a-teacher-in-england.md [hunk 2]
```diff
@@ -148,100 +148,12 @@ check if you intend to teach students under 18.
...
-To teach further education you do not need QTS. Instead you can study
-for a PGCE or PGDE without QTS. These courses are also eligible for
-candidates without a degree.
-
-You can also search for ‘further education’ courses leading to a post-compulsory education and training qualification (PCET). You do not need a degree to apply for courses leading to a PCET.
 
-### Choose an age group
+## Choose an age group
 
 You can train to teach:
```
Comment (peteryates) [inline content/guidance/become-a-teacher-in-england.md:158]: I guess there's a bit to add here?

**Review by peteryates — COMMENTED** [review]
Undoubtedly way-clearer than before. Again, a third pair of eyes to check nothing important's been omitted is a good idea.

File: content/guidance/become-a-teacher-in-england.md (hunk 2 above)
Comment (EmmaFrith) [reply content/guidance/become-a-teacher-in-england.md:158]: not sure i follow?

Comment (peteryates) [reply content/guidance/become-a-teacher-in-england.md:158]: ah my bad, it looks fine

**Review by MylesJarvis — APPROVED** [review]

**Review by peteryates — APPROVED** [review]

**Pull request merged**